network-api/networkapi/wagtailpages/templates/wagtailpages/blog_index_page.html

```diff
@@ -22,6 +22,15 @@
   {% endif %}
 {% endblock %}
 
+{% block featured_video_post %}
+  {% if not filtered %}
+    {%  for video_post in page.specific.featured_video_post.all %}
+      {% include "./fragments/blog_index_featured_video_post.html" with blog_page=video_post.blog_page video_url=video_post.video_url %}
+    {% endfor %}
+  {% endif %}
+
+{% endblock %}
```

Comment:
**Suggestion** The "featured video post" still fits the "featured" concept. Why not add it to the existing block above? You would not need to edit the base template in that case.

Reply:
Great suggestion Tibor, I updated it to be in the featured section as that makes more sense to me as well!